### DISCUSSION TITLE
Use Sioyek with most recent commits without building from source locally

### DISCUSSION THREAD

**Discussion by hasantahir** [ahrm/sioyek#337] [Q&A] [answered]
While the release cycle is fairly quick, wondering if there is any other way to try out the latest commits without having to build the app from the source? On my mac machine, I have had several unsuccessful tries building from the source. I get all sorts of compile errors that unfortunately I don't have the skills to resolve.

Thanks for this wonderful PDF reader.

Cheers

**Comment by ahrm** [comment, marked as answer]
I occasionally upload experimental builds here:
https://github.com/hexomancer/sioyek/releases/

**Reply by hasantahir** [reply]
Great. thanks.

**Reply by hasantahir** [reply]
@ahrm can you kindly push the latest updates to the experimental builds? Would love to try out the smooth scrolling feature.

Thanks.

**Reply by ahrm** [reply]
done.

**Reply by hasantahir** [reply]
Just tried the latest one and the `fit_to_page_xx` is not working. I can not change this view of the two-column PDF:

<img width="1747" alt="Screenshot 2022-09-21 at 12 30 06" src="https://user-images.githubusercontent.com/11236148/191493555-dd82b9f9-ba0f-4e93-86ab-7bd6738c2afe.png">
